### DISCUSSION TITLE
Is there a course deadline?

### DISCUSSION THREAD

**Discussion by theGuyWithBlackTie** [OpenMined/courses#198] [Project: Privacy Product Specification] [answered]
Hi.
So I am doing the first course and I am going a little slow as I've other universities' tasks, submissions which I am working on parallely. So I wanted to know is there any deadline to complete the course and the project?

Thanks.

**Comment by yemikifouly** [comment, marked as answer]
Hello Ashish, the guy with the black tie! 😀 I am glad to hear that you have started the course and are enjoying it so far. 🙂 There is no deadline for the course. Complete it at your own pace. 🙂
